libraries/chain/contracts/eosio_contract.cpp

```diff
@@ -505,11 +504,11 @@ void apply_eosio_postrecovery(apply_context& context) {
 
    context.execute_deferred(std::move(dtrx));
 
-
    auto data = get_abi_serializer().variant_to_binary("pending_recovery", record_data);
-   const uint64_t id = account;
+   const uint128_t id = account;
```

Comment:
Why is an account_name type being converted to an uint128_t to pass into db_find_i64 as a uint64_t?